test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,42 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        //empty list
+        assertDoNotHaveNull();
+
+        // one element( has null)
+        assertHasNull((Object) null); //one null object
+
+        // one element (no null)
+        assertDoNotHaveNull(1);
+        assertDoNotHaveNull("a");
+        assertDoNotHaveNull("");
+
+        // 2 elements one null object + one number
```

Comment:
These comment will be confusing for others when you have assertDoNotHaveNull together with assertHasNull below

Reply:
Noted. Thank you!